stylesheet.css

```diff
@@ -0,0 +1,3 @@
+.header {
+  color: #00f;
+}
```

Comment:
- [ ] From the project requirement you should style the `hello microverse!` with your favorite color. I can see you are writing this style but it is not affecting the  `hello microverse!` text in your `index.html` :smile: